### ISSUE TITLE
Items - The '# of items selected' text is not announced when changed

### ISSUE THREAD

**Issue by aviannamiller** [omeka/Omeka#1141]
"Description: On the 'Items' page, there is a button labeled 'Select all 16 results'. However, when screen reader users select it or deselect it, the screen reader fails to read the button's changed state (e.g. "16 items selected", "0 items selected"). This can confuse screen reader users if the items are selected or not, which can be devastating if they accidently select the 'Delete' button while everything is selected. The ""# items selected"" text is also not announced when the user selects any of the checkboxes.

Expected Results: Users should always know how many items are selected based off the amount of check boxes selected and should update the user on the current amount of items selected. This is typically done using aria-live regions.

Actual Results: Screen readers fail to read the # of selected items status update message."

4.1.3 Status Messages; Level AA; WCAG 2.1

**Comment by kimisgold** [comment]
There is a fix drafted on the [`a11y/announce-selected-items`](https://github.com/omeka/Omeka/tree/a11y/announce-selected-items) branch. Whenever the user manages items for selection on the browse view, screen readers should read out the contents of the "# items selected" status.

One thing I immediately noticed is that Voiceover reads it out of order: for example, "items selected 3" instead of "3 items selected". I'm using `aria-atomic` to read out the contents of that live region, and I suspect it's reading the unchanged content first. I'm not sure how consistent this is across screen readers or what to do about it for Voiceover.

**Comment by aviannamiller** [comment]
Same experience with VoiceOver, and here are the differing results with other tested screen reader / browser combos:

- JAWS/Firefox: "Items selected, 3 items selected"
- JAWS/Chrome/Edge: "3 items selected"
- NVDA/all browsers: "3 items selected"

My best guess would be that VO (in addition to JAWS/Firefox) has a different manner of handling the implicit `aria-relevant`, but I can't find anything noted to that effect in documentation of support.

**Comment by kimisgold** [comment]
I've changed the test to read as "Items selected: #". This gets us around the "1 items selected" case, and ensures Voiceover will read the string in the correct order.
